dev_docs/07_visual_testing_guide.md

```diff
@@ -26,123 +32,180 @@ KDS has a visual testing system that allows you to take snapshots of how KDS Com
      export PERCY_TOKEN=<your-percy-token>
      ```
 
-2. **Run Visual Tests**:
+2. **Run visual tests:**
 
-   - Execute the tests with the visual testing configuration:
+   - Execute the tests:
 
      ```bash
      yarn test:visual
      ```
 
-3. **Check results**:
-   - Head over to the Percy Dashboard and check the results for the latest Percy build.
+3. **Check results:**
 
-### Visual testing workflow
+   - Go to the Percy Dashboard and review the results for the latest Percy build.
 
-We use GitHub Actions to execute the visual testing workflow on every PR. When changes to a PR are made, a deployment request is initiated. Only **Learning Equality team members** can approve this request. Once approved, the deployment is executed, and visual tests are run automatically. The results of the test run are surfaced in the form of an automated comment containing a link to the Percy build.
+## Create visual tests for a component
 
-**Note:** Developers outside of Learning Equality can only review the visual changes for local test runs. To review the visual tests executions that run on GitHub Actions, one needs to be a part of Learning Equality's Browserstack team.
+Most commonly, each visual test scenario for a given component is placed as a `.vue` file in the [`/examples`](../examples) directory—a shared location for visual examples used in both documentation and visual testing. These example files are then imported into a single test page, `ComponentVisualTest.vue`, which is snapshotted by the component's Jest test suite.
 
-### Writing visual tests
+A visual test page for a component typically includes (1) examples from the documentation website that are suitable for visual testing and (2) additional scenarios, such as edge cases. **Always check if there are existing `/examples` that can be imported into visual tests instead of duplicating scenarios.**
 
-You can write the visual tests alongside the unit tests, i.e. in the same test file. Take a look at [`KButton.spec.js`](../lib/buttons-and-links/__tests__/KButton.spec.js) to familiarize yourself with writing visual tests.
+Let's see how this works in practice by writing visual tests for `KTextbox`:
 
-1. **Import Utility Functions**:
+1. **Create a page to contain all visual tests** for `KTextbox` in [`/lib/KTextbox/__tests__/components/KTextboxVisualTest.vue`](../lib/KTextbox/__tests__/components/KTextboxVisualTest.vue). This is the page that will be later snapshotted.
 
-   - Import the utility functions from [`visual.testUtils.js`](../jest.conf/visual.testUtils.js):
+2. **For each scenario you wish to test, check if there is a visual example in the `/examples` directory, or add a new one.** For example, [`/examples/KTextbox/WithLabel.vue`](../examples/KTextbox/WithLabel.vue) shows a simple textbox with label, and [`/examples/KTextbox/Validation.vue`](../examples/KTextbox/Validation.vue) shows textbox with validation.
```

Comment:
So by default the convention will be to create an example file for each component state? Even if they are not in the docs, right?